### PR TITLE
Use stable version

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Review by xabbuh — APPROVED** [review]

**Review by xabbuh — CHANGES_REQUESTED** [review]
the `composer.lock` file is now no longer up-to-date

File: composer.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
         "twig/markdown-extra": "^3.3"
     },
     "require-dev": {
-        "dama/doctrine-test-bundle": "^8.0.0-beta5",
+        "dama/doctrine-test-bundle": "^8.0.0",
         "doctrine/doctrine-fixtures-bundle": "4.0.x-dev",
```
Comment (rosier) [inline composer.json:53]:
```suggestion
        "doctrine/doctrine-fixtures-bundle": "^3.5",
```
Same as https://github.com/symfony/demo/pull/1457/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R53

Comment (OskarStark) [reply composer.json:53]: Why going back down to 3.5?

**Comment by OskarStark** [comment]
> the composer.lock file is now no longer up-to-date

done @xabbuh

**Review by xabbuh — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (rosier) [reply composer.json:54]: That's the latest stable

**Comment by javiereguiluz** [comment]
Closing because we already did this change as part of the upgrade to Symfony 7. Oskar, I'm sorry we didn't merge this on time.

**Pull request closed without merging**